src/content/docs/infrastructure/microsoft-azure-integrations/get-started/activate-azure-integrations.mdx

```diff
@@ -13,29 +13,57 @@ redirects:
   - /docs/infrastructure/azure-integrations/getting-started/activate-azure-integrations
   - /docs/infrastructure/microsoft-azure-integrations/getting-started/activate-azure-integrations
   - /docs/integrations/microsoft-azure-integrations/getting-started/activate-azure-integrations
-  - /docs/infrastructure/integrations/integrations/azure-integrations
+  - /docs/infrastructure/integrations/integrations/azure-integrations 
 ---
 
-New Relic [infrastructure integrations](/docs/infrastructure/integrations/intro-infrastructure-integrations) allow you to report data from specific systems and supplement [infrastructure's](/docs/infrastructure/new-relic-infrastructure/getting-started/introduction-new-relic-infrastructure) default, automatic monitoring. The Microsoft Azure integrations report data from various Azure platform services to your New Relic account. This document explains how to activate Azure integrations.
+For all our solutions for Azure monitoring, see [Introduction to Azure integrations](/docs/infrastructure/microsoft-azure-integrations/get-started/introduction-azure-monitoring-integrations). 
```

Comment:
Instead of starting off this doc with a link to another doc, why not put that content in a collapser a bit farther down